style.css

```diff
@@ -0,0 +1,107 @@
+body {
+font-family: Roboto;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)